hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -132,5 +124,119 @@
+           '''
+    }
+    {
+     name: ctrl_aux_cg
+     desc: '''
+           Covers when enabled a complete write forces a reseed.
+           this is done by checking the DUT goes out of idle state after a full key has been provided.
+           also covers that this is not the case then key_touch_forces_reseed = 0.
+           '''
+    }
+    {
+     name: regwen_cg
+     desc: '''
+           Covers that an attempt to write to the ctrl_aux was made when regwen is disabled.
+           '''
+    }
+    {
+     name: trigger_cg
+     desc: '''
+           This covergroup has two very different cover points.
+           - start covers that a start initiates an operation in manual mode.
+             and that it does not when not in manual mode
+           - that a write to key_iv_data_in/data_out_clear clear clears the data from the register
+            Additionally it covers that going from automatic mode to manual mode it is not possible to trigger a start without configuring the DUT (writing to CTRL should trigger a need for new configuration)
+           The prng reseed is covered by the reseed_cg
+           '''
+    }
+    {
+     name: status_cg
+     desc: '''
+           Covers the different status bits was seen
+           '''
+    }
+    {
+     name: reseed_cg
+     desc: '''
+           Cover that the different reseed configurations has been used.
+           '''
+    }
+    {
+     name: fault_inject_cg
+     desc: '''
+           Cover that a recoverable error has been seen:
+           - When the DUT is idle but just about to start
+           - When the DUT is busy
+           '''
+    }
+    {
+     name: self_clearing_cg
+     desc: '''
+           Cover that the DUT self clearing is working correctly.
+           An attack could be made by triggering an operation after a reset without configuring the DUT.
+           The self clearing mechanism should prevent the DUT from starting.
+           This mechanism should also clear any data in the output register with random data
+           After a reset is pulled two things will be covered
+           - manually write trigger.start and poll status.idle and make sure the DUT stays in idle.
+           - read output registers make sure output is no longer present
+           '''
+    }
+    {
+     name: dut_busy_cg
+     desc: '''
+           Cover that a busy DUT cannot be manipulated.
+           This includes:
+           - Reading output registers does not reveal internal states.
+           - Trying to change the configuration (CTRL)
+           - Trying to change the key
+           - Trying to change the IV
+           '''
+    }
+    {
+     name: sideload_cg
+     desc: '''
+           Cover sideload functionality
+           This includes:
+           - That an operation does not start before a valid key is present at the sideload interface with sideload enabled.
+           - That a key on the sideload interface is not consumed when sideload is disabled.
```

Comment:
perhaps also add a case that an operation with sideload enabled while the key isn't valid.

Reply:
that is the first bullet in this covergroup.
I don't think we need a dedicated testpoint for this. the sideload test point should test this  which will be verified by this covergroup